allensdk/brain_observatory/gaze_mapping/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = '1.0.0'
```

Comment:
I don't think that we should be specifying a version for this module separate from the allensdk as a whole. If it is different enough from the allensdk such that it warrants a separate development cycle and version then it should be its own repo.

Reply:
Using AllenSDK version to version the output files now!